Sources/Document/CRDT/CRDTArray.swift

```diff
@@ -123,7 +123,9 @@ extension CRDTArray {
      * `toJSON` returns the JSON encoding of this array.
      */
     func toJSON() -> String {
-        let json = self.elements.map { $0.getValue().toJSON() }
+        let json = self.elements
+            .filter { $0.getValue().isRemoved() == false }
```

Comment:
@hackerwins In my opinion, CRDTArray.toJSON() shouldn't have removed elements. Is that right?